src/docs/get-started/install/_ios-setup.md

```diff
@@ -152,6 +157,8 @@ to set up physical device deployment in Xcode.
 [Check the app's Bundle ID]: /images/setup/xcode-unique-bundle-id.png
 [Choosing a Membership]: https://developer.apple.com/support/compare-memberships
 [Mac App Store]: https://itunes.apple.com/us/app/xcode/id497799835
+[Flutter plugins]: /docs/development/packages-and-plugins/using-packages
```

Comment:
I wanted to link somewhere more specific that explained what plugins were (with native code), but I couldn't really find anything.  Filed https://github.com/flutter/website/issues/4661.

Reply:
Thanks for filing that bug, @jmagman. The info is a bit buried on the [developing packages](https://flutter.dev/docs/development/packages-and-plugins/developing-packages#types) page, but both could use some clarification.

Reply:
Ah ha, there it is!  Will update the link to that anchor.